tests/unittests/OperatorTest.cpp

```diff
@@ -7976,6 +7976,71 @@ TEST_P(OperatorTest, FusedRowwiseQuantizedSparseLengthsWeightedSum_Float) {
       bindings_, mod_, F_, EE_, ElemKind::UInt8FusedQTy, 0.0001);
 }
 
+/// Test Fused-RWQ-SLWS in where the weights are in Fp16, inputs are
+/// UInt8FusedQTy.
+TEST_P(OperatorTest,
+       FusedRowwiseQuantizedSparseLengthsWeightedSum_float16_weights) {
```

Comment:
nit: Can you capitalize `float16` so it's consistent with the rest of the tests?